backend/src/grpc.rs

```diff
@@ -0,0 +1,40 @@
+use tonic::{transport::Server, Request, Response, Status};
+
+use hello_world::greeter_server::{Greeter, GreeterServer};
+use hello_world::{HelloReply, HelloRequest};
+
+pub mod hello_world {
+    include!("../../common/codegen/helloworld.rs");
```

Comment:
let's add common as dependency to backend...

Reply:
Good point, much nicer with re-exports